specs/binding-grpc.spec/src/main/scripts/io/aklivity/zilla/specs/binding/grpc/schema/grpc.schema.patch.json

```diff
@@ -32,6 +32,7 @@
                         "enum": [ "server", "client"]
                     },
                     "vault": false,
+                    "options": false,
```

Comment:
now that the options services is deprecated there aren't any other options for the binding.

Reply:
Btw, i think by the same token we don't currently do routing by `grpc` service method on `grpc` `client` binding.

Reply:
This change broke some tests so I have reverted it. I will capture an issue to make these changes.